dlls/extdll.h

```diff
@@ -34,6 +34,12 @@
 
 #include "archtypes.h"     // DAL
 
+
+// Misc C-runtime library headers
+#include "stdio.h"
+#include "stdlib.h"
+#include "math.h"
+
```

Comment:
This seems to belong to a different commit.

Reply:
yep, linux compat or whatever, done